operations/services.md

```diff
@@ -455,10 +486,28 @@ Updates information about the specified [Availability block](#availability-block
-| `StartUtc` | [String update value](reservations.md#string-update-value) | required | Start of the interval in UTC timezone in ISO 8601 format \(or `null` if the start time should not be updated\). |
-| `EndUtc` | [String update value](reservations.md#string-update-value) | required | End of the interval in UTC timezone in ISO 8601 format \(or `null` if the end time should not be updated\). |
-| `ReleasedUtc` | [String update value](reservations.md#string-update-value) | required | The moment when the block and its availability is released \(or `null` if the released time should not be updated\). |
-| `ExternalIdentifier` | [String update value](reservations.md#string-update-value) | optional, max 255 characters | Identifier of the block from external system \(or `null` if the identifier should not be updated\). |
+| `StartUtc` | [String update value](reservations.md#string-update-value) | optional | Start of the interval in UTC timezone in ISO 8601 format \(or `null` if the start time should not be updated\). |
+| `EndUtc` | [String update value](reservations.md#string-update-value) | optional | End of the interval in UTC timezone in ISO 8601 format \(or `null` if the end time should not be updated\). |
+| `ReleasedUtc` | [String update value](reservations.md#string-update-value) | optional | The moment when the block and its availability is released \(or `null` if the released time should not be updated\). |
+| `ExternalIdentifier` | string [String update value](reservations.md#string-update-value) | optional, max 255 characters | Identifier of the block from external system \(or `null` if the identifier should not be updated\). |
+| `State` | string [Availability block state update value](#availability-block-state-update-value) | optional | State of the availability block \(or `null` if the state should not be updated\). |
+| `ReservationPurpose` | string [Reservation purpose update value](#reservation-purpose-update-value) | optional | Purpose of the reservation \(or `null` if the purpose should not be updated\). |
+| `CompanyId` | [String update value](reservations.md#string-update-value) | optional | Unique identifier of the [Company](enterprises.md#company) the block is assigned to \(or `null` if the company should not be updated\). |
+| `BookerId` | [String update value](reservations.md#string-update-value) | optional | Unique identifier of the [Customer](customers.md#customer) the block is assigned to \(or `null` if the booker should not be updated\). |
+| `Notes` | [String update value](reservations.md#string-update-value) | optional | Additional notes of the block \(or `null` if the notes should not be updated\). |
+| `Budget` | [Currency update value](finance.md#currency-update-value) | optional | The tentative budget for the total price of reservations \(or `null` if the budget should not be updated\). |
```

Comment:
For tomorrow discussion:
We do not specify it but we should start doing it - From the documentation, you have no idea that those fields can be deleted from the entity (set null). we should come up with some format how to say it in the documentation.